recipes/sml/all/conandata.yml

```diff
@@ -2,3 +2,7 @@ sources:
   "1.1.4":
     url: https://github.com/boost-ext/sml/archive/v1.1.4.tar.gz
     sha256: 897c78077f5a4d22b0de253b8689a8ea6d8adfba8d7b4877d6f5c76277e00976
+patches:
+  "1.1.4":
+    - patch_file: "patches/clang12.patch"
```

Comment:
We typically like to make patches more descriptive... like "0001-fix-clang12-error.patch" just as a suggestion

Also has this been submitted to upstream?

Reply:
> Also has this been submitted to upstream

There is an issue on the project github. "https://github.com/boost-ext/sml/issues/447"